api-specificatie/openapi.yaml

```diff
@@ -168,15 +168,15 @@ paths:
           $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/responses.yaml#/503'
         default:
           $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/responses.yaml#/default'
-  /ligplaatsen/{identificatie}:
+  /adresseerbaarobject/{identificatie}:
```

Comment:
Ik denk dat het goed is de parameter identificatie te hernoemen naar adresseerbaarobjectIdentificatie, zodat er duidelijk onderschied is met welke ID's er gezocht kan worden.

Reply:
Ligplaatsen als endpoint is verwijderd omdat de vraag was om het adresseerbaar object te ontsluiten. Daar is de path parameter in derdaad adresseerbaarObjectIdentificatie geworden. 

Ik wil het nut van de queryparameter adresString graag met Cathy bespreken. Ik ziet die in deze context niet, dus ik wil graag de ratio daarachter eerst bespreken.  (#44)